http_service/models.py

```diff
@@ -60,3 +65,60 @@ def retrieve_model(name):
         LOGGER.info(f"ETAG for {model_url} is ok")
 
     return file_path
+
+
+def classify_bug(
+    model_name, bug_ids, bugzilla_token, expiration=DEFAULT_EXPIRATION_TTL
+):
+    # This should be called in a process worker so it should be safe to set
+    # the token here
+    bug_ids_set = set(map(int, bug_ids))
+    bugzilla.set_token(bugzilla_token)
+    bugs = bugzilla._download(bug_ids)
+
+    redis_url = os.environ.get("REDISTOGO_URL", "redis://localhost/0")
+    redis = Redis.from_url(redis_url)
+
+    missing_bugs = bug_ids_set.difference(bugs.keys())
+
+    for bug_id in missing_bugs:
+        redis_key = f"result_{model_name}_{bug_id}"
+
+        # TODO: Find a better error format
+        encoded_data = json.dumps({"available": False})
+
+        redis.set(redis_key, encoded_data)
+        redis.expire(redis_key, expiration)
+
+    if not bugs:
+        return "NOK"
+
+    # TODO: Cache the model in the process memory, it's quite hard as the RQ
+    # worker is forking before starting
+    model = load_model(model_name)
+
+    # TODO: Classify could choke on a single bug which could make the whole
+    # job to fails. What should we do here?
```

Comment:
Maybe we could do the same as for missing bugs earlier?

Reply:
The issue is that we are giving X bug ids to `classify` and get an Exception in return so we don't know which one was failing

Reply:
Ah right, I remember we talked about that.